README.md

```diff
@@ -87,19 +87,19 @@ $ npm install restify
 
 Restify aims to support the Node.js LTS (Active and Maintenance) versions along with Node.js current stable version.
 
-| Node Release  | Supported in Current Version | Notes    |
-| :--:     | :---: | :---:       | 
-| 11.x | **Yes**      | Current stable | 
-| 10.x | **Yes**      | Active LTS | 
-| 8.x  | **Yes** | Maintenance LTS  | 
-| 6.x  | **No** | Use restify v7.x, team will backport critical issues only   | 
-| 4.x  | **No** | Use restify v7.x, team will backport critical issues only  | 
+| Node Release | Supported in Current Version | Notes |
+| :--: | :---:   | :---: |
+| 18.x | **No**  | Current stable. See [#1888](https://github.com/restify/node-restify/issues/1888) |
+| 16.x | **Yes** | Active LTS |
+| 14.x | **Yes** | Maintenance LTS |
+| 12.x | **No**  | Use restify v8.x, team will backport critical issues only |
+| 12.x | **No**  | Use restify v8.x, team will backport critical issues only |
```

Comment:
Duplicate line? unless you meant to say 10.x?

Reply:
removed, thanks.